### PR TITLE
MKL compatibility

### PR DESCRIPTION
**Pull request by ssskkkky**
we can use emme in kylin2 now! where only MKL exists.

### CODE REVIEW

File: include/solver.h [hunk 1]
```diff
@@ -81,9 +91,16 @@ class EigenSolver {
         //               rwork.data(), &info);
 
         // zgesdd is much faster than zgesvd
+	#ifdef EMME_MKL
```
Comment (12ff54e) [inline include/solver.h]: It seems like you formatter did not work properly here

File: include/solver.h [hunk 2]
```diff
@@ -127,9 +144,15 @@ class EigenSolver {
...
         Timer::get_timer().start_timing("linear solver");
+	#ifdef EMME_MKL
+	zsysv(upper, &dim, &dim, eigen_matrix.data(), &dim, ipiv.data(),
+                     eigen_matrix_derivative.data(), &dim, work.data(),
+                     &work_length, &info);     
+	#else
         LAPACK_zsysv(upper, &dim, &dim, eigen_matrix.data(), &dim, ipiv.data(),
                      eigen_matrix_derivative.data(), &dim, work.data(),
                      &work_length, &info);
+	#endif
```
Comment (12ff54e) [inline include/solver.h]: Same format issue above

**Review by 12ff54e — CHANGES_REQUESTED** [review]
Looks good, except some minor format issues

**Comment by ssskkkky** [comment]
reformated

**Review by 12ff54e — APPROVED** [review]

**Pull request merged**